src/test/tor/minimal/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ add_shadow_tests(BASENAME tor-minimal
                    --use-cpu-pinning true
                    --parallelism 2
                    --strace-logging-mode off
+                   # Disable to support fork
+                   --use-memory-manager false
```

Comment:
Should the `shadow_config_spec.md` docs also say the memory manager must be disabled for fork?